<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>changeshkins</class>
 <widget class="QWidget" name="changeshkins">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Microsoft YaHei UI</family>
    <pointsize>11</pointsize>
    <weight>75</weight>
    <bold>true</bold>
   </font>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>20</y>
     <width>72</width>
     <height>15</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <family>Microsoft JhengHei UI</family>
     <pointsize>11</pointsize>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>皮肤</string>
   </property>
  </widget>
  <widget class="QWidget" name="">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>50</y>
     <width>346</width>
     <height>229</height>
    </rect>
   </property>
   <layout class="QGridLayout" name="gridLayout">
    <item row="0" column="0">
     <widget class="QPushButton" name="pushButton_red">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="1">
     <widget class="QPushButton" name="pushButton_blue">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="0" column="2">
     <widget class="QPushButton" name="pushButton_pink">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="0">
     <widget class="QPushButton" name="pushButton_orange">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="1">
     <widget class="QPushButton" name="pushButton_yellow">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
    <item row="1" column="2">
     <widget class="QPushButton" name="pushButton_green">
      <property name="minimumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="maximumSize">
       <size>
        <width>110</width>
        <height>110</height>
       </size>
      </property>
      <property name="text">
       <string/>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
